qt/scientific_interfaces/ISISReflectometry/GUI/Runs/RunsView.h

```diff
@@ -123,6 +124,7 @@ class MANTIDQT_ISISREFLECTOMETRY_DLL RunsView
   QBasicTimer m_timer;
 
   RunsTableView *m_tableView;
+  BatchView *m_batchView;
```

Comment:
I don't think this is used any more? This and the forward declaration could be removed.